midi/midi/decode.py

```diff
@@ -18,25 +20,26 @@ class EventNote:
 
     def __init__(
             self,
-            velocity: int | float,
+            velocity: Union[int, float],
```

Comment:
Why this change?

Reply:
it turns out a component (don't remember which one) fails at complex structures to differentiate logical "|" from mathematical "|" (CI threw a few errors)